_data/sites.json

```
@@ -2861,6 +2861,16 @@
         ]
     },
 
+    {
+        "name": "developer.oxforddictionaries.com",
```

What about 
```suggestion
        "name": "Oxford Dictionaries API",
```
?

(Then you'll need to sort the entry again)